rest_framework/static/rest_framework/css/default.css

```diff
@@ -32,10 +32,6 @@ h2, h3 {
   margin-right: 1em;
 }
 
-ul.breadcrumb {
-  margin: 70px 0 0 0;
-}
-
 .breadcrumb li.active a {
```

Comment:
I was referring to the `.breadcrumb li.active a` block, which is directly associated with the `ul.breadcrumb` block that was moved.

I should have made that more clear as GitHub doesn't really point out where comments are attached to.